contrib/arrow-cmake/CMakeLists.txt

```diff
@@ -1,5 +1,22 @@
 set (CMAKE_CXX_STANDARD 17)
 
+set(ARROW_VERSION "6.0.1")
```

Comment:
We completely don't need it as we don't install Arrow .so.

Reply:
It is needed, or something equivalent, to generate the config.h with proper values. Otherwise it will fail to compile with errors like:
```
/mnt/ch/ClickHouse/contrib/arrow/cpp/src/arrow/config.cc:33:5: error: indirection requires pointer operand ('int' invalid)
    ARROW_VERSION,
```

Reply:
Ok.